frontend/src/lib/components/kpi.svelte

```diff
@@ -1,9 +1,18 @@
 <script lang="ts">
+  import { cn } from '$lib/utils';
+  import type { HTMLAttributes } from 'svelte/elements';
+
   export let title: string;
+
+  let className: HTMLAttributes<HTMLDivElement>['class'] = undefined;
+  export { className as class };
```

Comment:
why not `export let class: HTMLAttributes... = undefined;`
Remember it is called `class` in Svelte and not `className`

Reply:
We do this in other files as well. If I change it to your suggestion I get:

<img width="631" alt="image" src="https://github.com/madsthom/mmr-project/assets/2279254/fc53410a-a23f-4527-9c62-01943940a72b">

Reply:
Guess I don't know how to svelte